mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -17,7 +17,7 @@ export const AuthPage = () => {
 
   useEffect(() => {
     handleUserAuthState(setUser);
-    handleUserCredentials();
+    writeNewUserCredentialsToDataBase();
```

Comment:
You can write it shorter  - writeNewCredentialsToDb()

Reply:
it is not db